### PR TITLE
Cav 687

### PR DESCRIPTION
**Pull request by mar36tin**

### CODE REVIEW

File: app/uk/gov/hmrc/cipphonenumberverification/controllers/InternalAuthAccess.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cipphonenumberverification.controllers
+
+import uk.gov.hmrc.internalauth.client.{IAAction, Predicate, Resource, ResourceLocation, ResourceType}
+
+object InternalAuthAccess {
```
Comment (johno1985) [inline app/uk/gov/hmrc/cipphonenumberverification/controllers/InternalAuthAccess.scala:21]: can we make this a trait and then mixin?

**Review by johno1985 — COMMENTED** [review]
minor comment

File: app/uk/gov/hmrc/cipphonenumberverification/controllers/InternalAuthAccess.scala (hunk 1 above)
Comment (mar36tin) [reply app/uk/gov/hmrc/cipphonenumberverification/controllers/InternalAuthAccess.scala:21]: Why?

Comment (johno1985) [reply app/uk/gov/hmrc/cipphonenumberverification/controllers/InternalAuthAccess.scala:21]: It's what we are doing elsewhere, including your other PR. I believe using a mixin to 'mix in' behaviour is more appropriate than creating a singleton object

**Review by johno1985 — APPROVED** [review]

**Pull request merged**